### ISSUE TITLE
Use two-column Weight Configuration (instead of 3)

### ISSUE THREAD

**Issue by KuraFire** [sourcecred/sourcecred#2510]
This proposes some tiny changes to improve the UX of the current frontend Explorer. This ticket spans PRs in two repos: `sourcecred/sourcecred` itself, and our own `cred` repo for the SourceCred community.

- [ ] 1. Reorder the plugins list on `cred` to balance out the sizes of each plugin's configurations list: #TBD
- [ ] 2. Update WeightConfig.js to use grid `xs=6` sizes, instead of `xs=4`, to make two columns instead of three: #TBD
- [ ] 3. Add styling to the `h3` headings for the plugins: #TBD

### Before:
![SourceCred WeightConfig before](https://user-images.githubusercontent.com/33087/99012808-f9bd6280-2503-11eb-88aa-f385f01cf27b.png)

### After:
![SourceCred WeightConfig after](https://user-images.githubusercontent.com/33087/99012817-fe821680-2503-11eb-82bf-855e41677527.png)

**Comment by KuraFire** [comment]
Also, I cannot for the life of me figure out why, in the weight config, the word "Node" (in the <h4> for "Node weights") gets replaced with ReactNode, but not in the "Edge weights" heading. ¯\_(ツ)_/¯ 😂 

…but it seems worth fixing that, unless it's intentional (the code suggests it's not).

**Comment by KuraFire** [comment]
I'm not sure of how to properly added the two classes `MuiTypography-colorPrimary` and `MuiTypography-h5` to the `<h3>` for each plugin column, but @topocount said he could look at it so I'll leave this third step for him or someone else to tackle. :)

**Comment by decentralion** [comment]
(Just FYI / for context: we agreed in an earlier meeting that we're going to remove this UI when we switch to the new explorer dashboard (& CredRank))

**Comment by KuraFire** [comment]
Yeah, this is just a real quick fix to make the existing interface a little nicer and more intuitive until that time.
